pkg/traces/remotewriteexporter/exporter_test.go

```diff
@@ -85,6 +85,29 @@ type mockManager struct {
 	instance *mockInstance
 }
 
+// ApplyConfigs is used to batch configurations for performance instead of the singular ApplyConfig
+func (m *mockManager) ApplyConfigs(configs []instance.Config) instance.BatchApplyResult {
+	successful := make([]instance.Config, 0)
+	failed := make([]instance.BatchFailure, 0)
+	for _, v := range configs {
+		err := m.ApplyConfig(v)
+		if err != nil {
+			failed = append(failed, instance.BatchFailure{
+				Err:    err,
+				Config: v,
+			})
+		} else {
+			successful = append(successful, v)
+		}
+	}
+	return instance.BatchApplyResult{
+		Failed:         failed,
+		Successful:     successful,
+		NonConfigError: nil,
+	}
+
+}
```

Comment:
Is it possible for the remotewriteexporter to use the existing mock manager in the instance package?

Reply:
It uses some non exported fields, which could probably be rewritten.  Would rather do in a separate PR.